### PR TITLE
Related articles

### PR DESCRIPTION
**Pull request by auddin431**
Added "Related Articles" feature at the bottom of each article

### CODE REVIEW

File: src/pages/podcasts/[slug].page.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
+import * as React from "react";
```
Comment (WagleTanvi) [inline src/pages/podcasts/[slug].page.tsx:1]: Remove podcasts

File: src/pages/podcasts/index.page.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
+import * as React from "react";
```
Comment (WagleTanvi) [inline src/pages/podcasts/index.page.tsx:1]: Remove podcasts

File: src/components/RelatedArticles.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+import { imgix, formatDateAbbreviated } from "../utils";
+import { GetArticle, GetArticles } from "../aws";
+
+export function RelatedArticles({
+  article,
+  articles,
+  classNames,
+  theme,
+}: {
+  article: GetArticle;
```
Comment (WagleTanvi) [inline src/components/RelatedArticles.tsx:14]: Just for better clarity might be good to name this parameter currentArticle or something of that nature. Also, a quick one sentence blurb at the top about this component just to improve documentation.

File: src/components/RelatedArticles.tsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  const currentArticleID = article.id;
+  const filteredArticles = articles.items[0].articles.filter(
+    (art) => art.id !== currentArticleID
+  );
+  // const article1 = filteredArticles[0];
+  // const article2 = filteredArticles[1];
+  // const article3 = filteredArticles[2];
+  // const article4 = filteredArticles[3];
+
+  return filteredArticles.length < 4 ? (
```
Comment (WagleTanvi) [inline src/components/RelatedArticles.tsx:28]: Great error checking.

**Review by WagleTanvi — CHANGES_REQUESTED** [review]
Only thing blocking from merging is the removal of podcasts. Other comments are just documentation.

**Review by auddin431 — COMMENTED** [review]
changed article to currentArticle, removed podcasts, and added some documentation

**Review by WagleTanvi — APPROVED** [review]
Lgtm

**Pull request merged**